.circleci/config.base.yml

```diff
@@ -1300,6 +1300,9 @@ commands:
                 command: |
                   source .circleci/local_publish_helpers.sh
                   source $BASH_ENV
+                  startLocalRegistry "$(pwd)/.circleci/verdaccio.yaml"
```

Comment:
Is this needed for adding test ?

Reply:
yes, see my comment above ^

Reply:
Btw. I was hesitating to add this across the board. But the dent on run time is `0.05 s` . So it seems that trying to being precise and do this in just one job is not worth the effort.